wildfly-elytron/pom.xml

```diff
@@ -401,7 +415,7 @@
         <dependency>
             <groupId>org.wildfly.common</groupId>
             <artifactId>wildfly-common</artifactId>
-            <!-- scope is compile ELY-1153 -->
+            <scope>provided</scope>
```

Comment:
This should still be commented: if this JAR is made "provided" then nothing downstream will work unless they add it explicitly.  It might be a better idea to leave this compile-scoped and then add a filter to the shade plugin.